plasma/plasma-mvp/specifications/morevp.md

```diff
@@ -0,0 +1,277 @@
+
+Note that this challenge means it’s possible for an honest user to lose `exit bond` as they might not be aware their transaction is non-canonical. We address this attack vector and several mitigations in detail later.
+
+
+#### Joining an Exit
+
+As noted earlier, it’s possible that some participants in a transaction may not be aware that the transaction is non-canonical. Owners of both inputs and outputs to a transaction may want to start an exit in the case that they would receive the funds from the exit. However, we want to avoid the gas cost of repeatedly publishing and proving statements about the same transaction. We therefore allow owners of inputs or outputs to a transaction to join (“piggyback”) an existing exit that references the transaction. This effectively “caches” the result of the “exitable” and “canonical” checks. 
+
+Users must join an exit within the first period. To join an exit, an input or output owner places a bond, `piggyback bond`. This bond is used to cover the cost of challenges that show the input or output is spent. A successful challenge blocks the specified input or output from exiting. These challenges must be presented before the end of the second period. 
+
+Note that it isn’t mandatory to piggyback an exit. Users who choose not to join an exit are choosing not to attempt a withdrawal of their funds. If the chain is byzantine, not piggybacking could potentially mean loss of funds.
+
+
+### Attack Vectors and Mitigations
+
+#### Honest Exit Bond Slashing
+
+It’s possible for an honest user to start an exit and have their exit bond slashed. This can occur if one of the inputs to a transaction is malicious and signs a second transaction spending the same input. 
+
+The following scenario demonstrates this attack: 
+1. Mallory spends `UTXO1` in `TX1` to Bob, creating `UTXO2`.
+2. `TX1` is in-flight.
+3. Operator begins withholding blocks while `TX1` is still in-flight.
+4. Bob starts an exit referencing `TX1` and places `exit bond`.
+5. Mallory spends `UTXO1` in `TX2`.
+6. In period 1 of the exit for `TX1`, Mallory challenges the canonicity of `TX1` by revealing `TX2`.
+7. No one is able to respond to the challenge in period 2, so `TX1` is determined to be non-canonical.
+8. After period 2, Mallory receives Bob’s `exit bond`.
+
+Mallory has therefore caused Bob to lose `exit bond`, even though Bob was acting honestly. We want to mitigate the impact of this attack as much as possible so that this does not prevent users from receiving funds. 
+ 
+
+##### Mitigations
+
+###### Bond Sharing
```

Comment:
move this after small exit bond as small exit bond directly relates to the mallory bob situation above

Reply:
All of these mitigations relate to the mallory/bob situation above.